src/libs/odrl/constraints.js

```diff
@@ -1,20 +1,3 @@
-export class Constraint {
```

Comment:
Finally, after all these years, the moment has come... to delete this awful piece of code. (whoever wrote it lol)